src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php

```diff
@@ -232,6 +234,8 @@ protected function renderView($view, array $parameters = array())
     protected function render($view, array $parameters = array(), Response $response = null)
     {
         if ($this->container->has('templating')) {
+            @trigger_error('Using the "templating" service is deprecated since version 3.3 and will be removed in 4.0. Use Twig instead.', E_USER_DEPRECATED);
```

Comment:
we should avoid enabling the component in the SE otherwise this will happen by default, https://github.com/symfony/symfony-standard/pull/861 does it but is old

Reply:
https://github.com/symfony/symfony-standard/pull/1110